nservicebus/azure-storage-queues/multi-storageaccount-support_aliases_asq_[7,].partial.md

```diff
@@ -0,0 +1,24 @@
+## Using aliases for connection strings to storage accounts for Scale Out
+
+In order to prevent accidentally leaking connection string values, it is recommended to use aliases instead of raw connection strings. When applied, raw connection string values are replaced with registered aliases removing the possibility of leaking a connection string value. The concept of using aliases for connection strings to storage accounts has been introduced in `NServiceBus.Azure.Transports.WindowsAzureStorageQueues` Version 7. When using a single account, aliasing connection string is limited to just enabling it. When Scaling Out is applied, it requires registering an alias for each storage account.
```

Comment:
It has been removed in https://github.com/Particular/docs.particular.net/pull/1926/commits/a664585cb600ae7e6f7af146f81bbfcc8de4afde and it does not contain it any longer. Am I missing something?